### DISCUSSION TITLE
👋 Velkommen til Jøkul

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2856] [Annet]
## ☝️ To kjappe husregler

Siden dette kanskje er første gang du er innom oss på GitHub har vi to husregler du burde kjenne til:
1. Prosjektet er offentlig synlig, så du kan ikke dele person-, forretningskritiske-, eller andre konfidensielle opplysninger her.
2. Vi har noen [etiske prinsipper og regler](https://github.com/fremtind/jokul/blob/main/CODE_OF_CONDUCT.md) som vi ber om at du følger.
  

## 🤔 Hva brukes Discussions til?

Discussions er ganske fritt, du kan starte en tråd på det aller meste. Nye "ting" eller endringer på designsystemet begynner ofte her. _(feks: nye mønstre, komponenter, egenskaper, etc)._ Dette er også en fin plass å spørre om hjelp eller innspill, eller vise fram noe du har jobbet med.

Vi har noen kategorier og merkelapper du kan bruke for å legge til ekstra kontekst. Om noen fra [kjerneteamet](https://jokul.fremtind.no/kom-i-gang/hjelp#kjerneteamet) ser at tråden hører hjemme i en annen kategori eller trenger en merkelapp, så ordner de det for deg.

## 🤷🏻‍♂️ Hva passer i Discussions, og hva går andre steder?

Det korte svaret er; _**nesten alt**_. Vi har ingen føringer for hvordan innlegg skrives eller hva de må inneholde, om du står fast og trenger tips så kan du [sjekke ut dette innlegget](https://github.com/fremtind/jokul/discussions/2390) eller ta en kikk på hva andre har skrevet tidligere.

For at vi skal kunne holde styr på alt som skjer er det fint at behovene ligger samlet på ett sted, særlig når det jobbes med mange ting _(i mange forskjellige team)_ samtidig. Siden de som kommer til å hjelpe deg med tingen din kanskje ikke er de samme folka som du jobber med i det daglige, er det fint at de kan lese seg opp på det du har tenkt. Derfor ønsker vi at at mest mulig skjer i Discussions. Noen ganger diskuterer vi ting som **_ikke kan deles offentlig_** (første husregel). Da er det interne kanaler _(Teams, etc)_ som gjelder.

Om du trenger oppmerksomheten til noen raskt, lag en diskusjonstråd som vanlig og send den som en lenke på Teams. Det er fort gjort at diskusjonen fortsetter på Teams hvis den starter der, og da er det ekstra viktig at innsikten oppsummeres på GitHub så ikke den blir borte.

## 📣 Sett opp varsler

Det er mye som skjer i hverdagen og mye å følge med på når man jobber i en stor organisasjon. For å gjøre det litt enklere for deg selv å henge med i svingene anbefaler vi at du setter opp noen _enkle_ varslinger. Det gjør du ved å trykke på _Watch_ øverst til høyre på denne siden.

![For å bare følge med på Discussions velger du Custom i menyen som dukker opp, huker av for Discussions. Er du utvikler? Sett deg opp som watcher for issues og pull requests også, så får du med deg hva som skjer på kodesiden av Jøkul.](https://user-images.githubusercontent.com/4263132/166209675-5f93b57f-4a80-4af7-b3a1-06992ab2e8cc.png)

_Varslinger: For å bare følge med på Discussions velger du **Custom** i menyen som dukker opp, huker av for Discussions. Er du utvikler? Sett deg opp som watcher for **issues** og **pull requests** også, så får du med deg hva som skjer på kodesiden av Jøkul._

## 🙋 Introduser deg selv
Vi vil gjerne bli bedre kjent med deg! Introduser deg selv ved å svare på denne posten. Hva heter du, og hva liker du å drive med? Har du noe du vil skryte av eller et kult tips å dele? Og hva er favorittgrønnsaken din?

**Comment by leon-ho** [comment]
Vær hilset 👋 Jeg heter Leon, skal jobbe som interaksjonsdesigner-vikar for Kari på Flyt til ca. feb/mars 2023. Liker å reise 🗾 og favorittgrønnsak: 🍟(🥔)

Tips som er spesielt aktuelt i dag: Sjekk om Figma er fiksa https://status.figma.com/

**Reply by wkillerud** [reply]
Velkommen! 👋  Og erstatt "Compiling" med "Figma er nede" 😄  https://xkcd.com/303/
